python/audit/cases/common.py

```diff
@@ -83,17 +83,24 @@ def get_obj_name(obj: Model, obj_type: str) -> str:
     return obj_name
 
 
-def get_or_create_audit_obj(object_id: str, object_name: str, object_type: str) -> AuditObject:
+def get_or_create_audit_obj(
+        object_id: str,
+        object_name: str,
+        object_type: str,
+        action_id: int | None = None,
+) -> AuditObject:
     audit_object = AuditObject.objects.filter(
         object_id=object_id,
         object_type=object_type,
+        action_id=action_id,
```

Comment:
Since `action_id` can be `null=True`, there is a chance to get 2 entries for one object

Reply:
refactored